doc/source/index.rst

```diff
@@ -5,8 +5,15 @@
 
 Welcome to Renormalizer's documentation!
 ========================================
-Renormalizer is a python package based on tensor network states for electron-phonon
-quantum dynamics.
+Renormalizer is a quantum dynamics package based on density matrix
+renormalization group (DMRG) and its time dependent formulation written in
+Python. All (TD-)DMRG algorithms are implemented within the framework of matrix
+product states (MPS) and matrix product operators (MPO). The package aims to
+provide a light-weight and efficient platform for quantum dynamics simulation
```

Comment:
The package is not really light-weight 😂 . I'd prefer "a powerful and efficient platform". After all, it's our "aim".

Reply:
Compared to other giant packages, such as heidelberg mctdh, Renormalizer is light-weight. "a powerful and efficient platform" is also Ok.

Reply:
As far as I know (from Xinxian) the Heidelberg mctdh is big due to its bad architecture. A known issue is that more than 6 layers are not supported because the logics are hardcoded. 
The Itensor package contains about 80k lines of C++ code and Renormalizer contains 10k Python code. We can almost say the "weight" of the package is almost the same with Itensor considering the expressiveness of Python and the extremely low code redundancy in Renormalizer (thanks to constant refactoring).

Reply:
Ok, I will modify the description.